pymongo/server_selectors.py

```diff
@@ -105,24 +107,24 @@ def writable_server_selector(selection: Selection) -> Selection:
     )
 
 
-def secondary_server_selector(selection):
+def secondary_server_selector(selection: Selection) -> Selection:
     return selection.with_server_descriptions(
         [s for s in selection.server_descriptions if s.server_type == SERVER_TYPE.RSSecondary]
     )
 
 
-def arbiter_server_selector(selection):
+def arbiter_server_selector(selection: Selection) -> Selection:
     return selection.with_server_descriptions(
         [s for s in selection.server_descriptions if s.server_type == SERVER_TYPE.RSArbiter]
     )
 
 
-def writable_preferred_server_selector(selection):
+def writable_preferred_server_selector(selection: Selection) -> Selection:
     """Like PrimaryPreferred but doesn't use tags or latency."""
     return writable_server_selector(selection) or secondary_server_selector(selection)
 
 
-def apply_single_tag_set(tag_set, selection):
+def apply_single_tag_set(tag_set: Mapping[str, Any], selection: Selection) -> Selection:
```

Comment:
Could you add type aliases for TagSet/TagSets as `Mapping[str, Any]`/`Sequence[TagSet]`?

Reply:
yes! done!